fetchai/ledger/api/bootstrap.py

```diff
@@ -0,0 +1,35 @@
+import requests
+
+
+def server_from_name(server_name):
+    # Get list of active servers
+    servers_response = requests.get('https://bootstrap.fetch.ai/networks/')
```

Comment:
I would try and do as much filtering on the backend as possible. So it would be good to change this to:

```suggestion
    params = {'active': 1, 'version': '<insert library version here>'}
    servers_response = requests.get('https://bootstrap.fetch.ai/networks/', params=params)
```

Reply:
(the active flag is most important here

Reply:
I'm not sure what we can do with the version string here - the api is specifying a version range (e.g.: 0.10.0 > x >= 0.9.0), and the network is also specifying a range (e.g.: 0.7.0 > x >= 0.6.0). The most sensible check I can think of is to see if the network version range falls entirely within that specified by the api - thoughts?

Reply:
For the moment I would take the current version say `0.8.1` and chop it down to `0.8` and post that as a requirement. I think that should work with this API